package.json

```diff
@@ -112,6 +112,7 @@
     "gifshot": "^0.4.5",
     "lodash": "^4.17.14",
     "mathjs": "^6.6.4",
-    "moment": "^2.24.0"
+    "moment": "^2.24.0",
+    "wavesurfer.js": "^3.3.3"
```

Comment:
Can you please test this feature thoroughly (as you update the version from 2.2.1 to 3.3.3)?

Reply:
The translation was being played correctly. What other things do I need to test?

Reply:
Slider bar, rendering, check the console, wave for small audio wave for big audio (intime or size). etc.

Reply:
Actually I’m not sure about the expected behaviour. Can you do that please?

Reply:
Btw, can you set it to `"3.3.3"` without `^` so that it doesn't get updated automatically and the coherence with the type definitions is kept.

Reply:
Done